test.js

```diff
@@ -0,0 +1,5 @@
+const AWS = require('aws-sdk');
+
+console.log(AWS.DynamoDB.Converter.output({'NS': [1,2,3]}));
+console.log(new Set([1,2,3]));
+console.log(AWS.DynamoDB.Converter.input([1,2,3]));
```

Comment:
Why is this file in the root directory?

Reply:
Wasn't sure if I was gonna work on this PR further before merging. So I kept it for now as a test file.